### ISSUE TITLE
npm install tries to move non-existing files

### ISSUE THREAD

**Issue by vsund** [blockstack/blockstack-browser#654]
When I try to run `npm install` it fails because it tries to move non-existing files in `blockstack.js`.

```
npm ERR! path /home/me/blockstack/blockstack.js/node_modules/dashdash/node_modules/assert-plus
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall rename
npm ERR! enoent ENOENT: no such file or directory, rename '/home/me/blockstack/blockstack.js/node_modules/dashdash/node_modules/assert-plus' -> '/home/me/blockstack/blockstack.js/node_modules/dashdash/node_modules/.assert-plus.DELETE'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```
I could get around this with creating `node_modules/dashdash/node_modules/.assert-plus`.

After that I got the same error with another file:
```
npm ERR! path /home/me/blockstack/portal/node_modules/.staging/assert-plus-62888c76
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall rename
npm ERR! enoent ENOENT: no such file or directory, rename '/home/me/blockstack/portal/node_modules/.staging/assert-plus-62888c76' -> '/home/me/blockstack/blockstack.js/node_modules/dashdash/node_modules/assert-plus'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```
Creating `node_modules/.staging/assert-plus-62888c76` didn't help. NPM removes it whilst the installation.

cc @larrysalibra

**Comment by larrysalibra** [comment]
@vsund I wasn't able to replicate this. Can you see if you still have this problem with the fix from #653 ?

**Comment by vsund** [comment]
See #653, also fixed this issue.
